src/utils/next.ts

```diff
@@ -74,6 +75,13 @@ export const scaffold = (wrapped : ScaffoldedGetServerSideProps, options? : Scaf
             ctx.z.setTokenData(reqWithSession.session.tokenData);
         }
 
+        try {
+            ctx.user = await ctx.z.resource('users', 'me').get();
```

Comment:
Please extract the actual user data object and store that as `ctx.user`, not the entire response object returned from `get()`.